lxd/api_cluster.go

```diff
@@ -288,6 +305,10 @@ func clusterPut(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(fmt.Errorf("ServerName must be empty when disabling clustering"))
 	}
 
+	if req.ServerName != "" && strings.HasPrefix(req.ServerName, "@") {
```

Comment:
Please can we have a constant for `@` that can be used here and anywhere else there is logic to check it starts with an `@` so we can see A) where the concept is used (as searching for "@" is tricky) B) that it shows to the reader the "@" means "group name prefix" or something like that?